### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by cjdenio**
This PR does a good bit of refactoring ♻️

- Changed RPC payload to be array of Dokku arguments (e.g. `["ps:restart", "haas"]`
- Switched to `cmd.Output` as it returns stdout/stderr in a convenient way
- Fixed error handling in dokkud
- Removed an unnecessary interface in the RPC client

### CODE REVIEW

File: dokkud/main.go [hunk 1]
```diff
@@ -4,70 +4,50 @@ import (
...
+		}
 	}
 
-	// log.Printf("Output: %+v\n", output)
+	output := string(stdout)
 
 	err = reply(ctx, output, nil)
 	if err != nil {
-		return err
+		reply(ctx, nil, err)
```
Comment (anirudhb) [inline dokkud/main.go]: Don't get the point of this change... if reply fails it's likely fatal :) just return the err

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm, small nit

File: dokkud/main.go (hunk 1 above)
Comment (cjdenio) [reply dokkud/main.go]: oh wait, i'm dumb

**Review by anirudhb — APPROVED** [review]
lgtm

**Pull request merged**